src/org/eclipse/swt/accessibility/AccessibleFactory.d

```diff
@@ -95,7 +95,7 @@ class AccessibleFactory {
         }
     }
 
-    private this (int /*long*/ widgetType) {
+    private this (int widgetType) {
```

Comment:
They are actually int (gint or GLint or int of Xlib).

I checked a declaration of all function in libraries. And if it is to should be AAA, I replaced it to int.